Fix bugs (#199)
AY2223S1-CS2103T-T11-2/tp: docs/UserGuide.md

<div markdown="block" class="alert alert-info">

**:information_source: Note about editing the `status` and `remark` field:**<br>

> The status and remark of the application can only be edited via the [`status` command](https://github.com/AY2223S1-CS2103T-T11-2/tp/blob/master/docs/UserGuide.md#updating-status-of-an-internship-application--status)

Reviewer: Please use a link to within the page, not to the github repo.
```
[`status` command](#updating-status-of-an-internship-application--status)
``` 
should work.

Format: `status INDEX STATUS`, where `STATUS` must be either `p`, `o` or `r`

| Parameter | Representation                             | Constraints                                                 |
|-----------|--------------------------------------------|-------------------------------------------------------------|

> | `INDEX`   | The index of the internship                | Must be an unsigned positive number                         |       |

Reviewer: ```suggestion
| `INDEX`   | The index of the internship                | Must be a positive unsigned integer                         |       |
```

Author: Thank you for pointing this out
